strobogrammatic.py

```diff
@@ -1,23 +1,29 @@
 import unittest
 
+
 class solution:
+
+
 	def is_strobogrammatic(self, num):
-		turn_map = {'9':'6', '6': '9', '1':'1', '8':'8'}
+		_TURN_MAP = {'9':'6', '6':'9', '1':'1', '8':'8', '0':'0'}
 		i, j = 0, len(num) - 1
 		while i < j:
-			if num[i] not in turn_map or num[j] != turn_map[num[i]]:
+			if num[i] not in _TURN_MAP or num[j] != _TURN_MAP[num[i]]:
 				return False
 			i += 1
 			j -= 1
 		return True
 
 class solution_test(unittest.TestCase):
+
+
```

Comment:
Two blank lines before the class def not after

                  return True
// blank one
// blank two
class solution_test